test/parallel/test-util-types.js

```diff
@@ -291,3 +292,9 @@ for (const [ value, _method ] of [
   assert.ok(!types.isCryptoKey());
   assert.ok(!types.isKeyObject());
 }
+
+builtinModules.forEach((module) => {
+  assert.ok(types.isBuiltinModule(module));
+});
+
+['fs', 'foo'].forEach((module) => assert(types.isBuiltinModule(module), false));
```

Comment:
`'fs'` should definitely return `true`.

Reply:
It would probably be a good idea to test inputs of different value types (e.g. non-string) to make sure the function doesn't throw or return unexpected results in those cases.

Reply:
Agree. I am building node on codespace and it is running out of memory. Also noticed the builds are failing.